candidates/models/needs_review.py

```diff
@@ -0,0 +1,70 @@
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.core.cache import cache
+from django.utils.translation import ugettext as _
+
+from popolo.models import Person
+
+# This is the number of the first edits of a user that are considered
+# to need review.
+NEEDS_REVIEW_FIRST_EDITS = 3
```

Comment:
Yeah, now that you say that, I agree, but I might leave that until a later PR if that's OK...